gotchas/dhis2-release-lifecycles.md

```diff
@@ -0,0 +1,91 @@
+# DHIS2 release lifecycles
+
+This is a walk through on the DHIS2  software releases and its lifecycles. Starting off we are gonna communicate some basic concepts in order to build a common language that we will use to better explain the release cycle.
+
+
+#### Backend core application 
+
+The first one is the backend service. The DHIS2 organisation has a single backend service which serves all the different frontend and mobile applications. Among other responsibilities the backend is also responsible for building the backend services and frontend core applications. The output of the build is a `war` file which the end users can download from our [download page](https://www.dhis2.org/downloads). 
+
+To communicate releases the [repository of the backend application](https://github.com/dhis2/dhis2-core) follows a branching naming convention. According to this convention the `master` branch includes the latest of the implemented software and is always the next candidate for a major release. For each major released version of the DHIS2 software there is a branch that follows the` 2.{x}` pattern. Eg. `2.35`. 
+
+
+#### Frontend core applications
+
+DHIS2 implements and maintains different core web applications.  The core web apps also follow a branching convention, however that is slightly different from the one followed in the backend. Same as in the backend the `master` branch includes the latest of the implemented software and is always the next candidate for a major release. However for each released version of the software there is a branch that follows the `v{x}` pattern. Eg. `v35`.  
+
+_Note: There are some core frontend apps that are currently now following this convention. While writing this the exceptions are the [data-visualiser-app](https://github.com/dhis2/data-visualizer-app/) which is using a different convention and [a list of apps ](https://github.com/dhis2/dhis2-core/tree/master/dhis-2/dhis-web/dhis-web-apps) where feature toggling is being used._
```

Comment:
typo - should be "... currently **not** following ..."

Reply:
good catch Phil. thanks